### DISCUSSION TITLE
Renovate does not respect checksum validation when updating Maven wrappers

### DISCUSSION THREAD

**Discussion by clayburn** [renovatebot/renovate#33227] [Request Help]
### How are you running Renovate?

A Mend.io-hosted app

### Please tell us more about your question or problem

Renovate does not properly apply Maven wrapper updates when the Maven wrapper is configured to use [checksum verification](https://maven.apache.org/wrapper/#checksum-verification-of-downloaded-binaries). There are two cases in which Renovate breaks:

* PRs to update the Maven distribution will not update the checksum of the Maven distribution. See [here](https://github.com/clayburn/renovate-testing/pull/5) for an example.
* PRs to update the Maven wrapper will remove the checksums for both the Maven wrapper and the distribution, disabling checksum verification. See [here](https://github.com/clayburn/renovate-testing/pull/6) for an example.

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by viceice** [comment]
checksums are not yet supprted

**Comment by rarkins** [comment]
Converted to issue: https://github.com/renovatebot/renovate/issues/33444
